Jenkins_jobs/WorkspaceInfo.groovy

```diff
@@ -158,10 +124,47 @@ def beautify(machineName, workspaceDirectory, workspaceStats, subdirectories) {
         }
     }
     output += "\n=======================================================================\n\n";
+    return output
 }
 
-
-timeout(time: params.timeout, unit: 'HOURS') {
-    parallel clones
+def errorMessage(String machineName, String message) {
+    String errorOutput = "\n\n****NEEDS_ATTENTION****"
+    errorOutput += "\n${machineName} failed"
+    errorOutput += "\nMessage:\n${message}"
+    errorOutput += "\n****NEEDS_ATTENTION****\n\n"
+    return errorOutput
 }
 
+def setupParallelPipelines(
+        timeOut,
+        machineName,
+        workspaceDirectory,
+        workspaceStatscmd,
+        subdirectoryStatscmd,
+        statOutputHeading
+        ) {
+
+    String currentMachine = env.NODE_NAME
+    clones[machineName] = {
+        try {
+            timeout(time: timeOut.getTime(), unit: 'HOURS') {
```

Comment:
I don't think we need all this fancy timeout stuff, I think we can just ref the param here